src/commands/deployment/mark.ts

```diff
@@ -0,0 +1,73 @@
+import {Command, Option} from 'clipanion'
+
+import {TagCommand} from '../tag/tag'
+
+import {CUSTOM_TAGS_TAG, ENV_TAG, IS_DEPLOYMENT_TAG, IS_ROLLBACK_TAG, REVISION_TAG} from './constants'
+
+/**
+ * This command is a wrapper around the datadog-ci tag command, allowing customers to mark CI Jobs
```

Comment:
nit: (to align with https://github.com/DataDog/datadog-ci/pull/1141#discussion_r1425524579)
```suggestion
 * This command is a wrapper around the datadog-ci tag command, allowing customers to mark CI jobs
```